doc/transform_methods/hansenlaw.rst

```diff
@@ -141,13 +132,71 @@ Historical Note
+
+  .. math:: 
+
+    \tilde{A} = \rm{diag}[\lambda_1, \lambda_2, ..., \lambda_K]
+
+    \tilde{B} = [h_1, h_2, ..., h_K]^T
+
+    \tilde{C} = [1, 1, ..., 1]
+
+The differential equations have the transform solutions, forward:
+
+ .. math:: x(r) = \Phi(r, r_0) x(r_0) + 2 \int_{r_0}^{r} \Phi(r, \epsilon) \tilde{B} f(\epsilon) d\epsilon.
+
+and, inverse:
+
+ .. math:: x(r) = \Phi(r, r_0) x(r_0) - \frac{1}{\pi} \int_{r_0}^{r} \frac{\Phi(r, \epsilon)}{r} \tilde{B} g^\prime(\epsilon) d\epsilon,
+
+
+with :math:`\Phi(r, r_0) = \rm{diag}[(\frac{r_0}{r})^{\lambda_1}, ..., (\frac{r_0}{r})^{\lambda_K}] \equiv \rm{diag}[(\frac{n}{n-1})^{\lambda_1}, ..., (\frac{n}{n-1})^{\lambda_K}]`, where the integration limits :math:`(r, r_0)` extend across one grid interval or a pixel, so :math:`r_0 = n\Delta`, :math:`r = (n-1)\Delta`.
+
+To evaluate the (superposition) integral, the driven part of the solution, the
+driving function :math:`f(\epsilon)` or :math:`g^\prime(\epsilon)` is assumed to
+either be constant across each grid interval, the **zero-order hold** approximation, :math:`f(\epsilon) \sim f(r_0)`, or linear, a **first-order hold** approximation, :math:`f(\epsilon) \sim p + q\epsilon = (r_0f(r) - rf(r_0))/\Delta + (f(r_0) - f(r))\epsilon/\Delta`. The integrand then separates into a sum over terms multiplied by :math:`h_k`, 
+
+ .. math::
+
+    \sum_k h_k f(r_0) \int_{r_0}^{r} \Phi_k(r, \epsilon) d\epsilon
+
+with each integral:
+
+ .. math::
+
+  \int_{r_0}^{r} \left(\frac{\epsilon}{r}\right)^\lambda_k d\epsilon = \frac{r}{r_0}\left[ 1 - \left(\frac{r}{r_0}\right)^{\lambda_k + 1}\right] = \frac{(n-1)^a}{\lambda_k + a} \left[ 1 - \left(\frac{n}{n-1}\right)^{\lambda_k+a} \right],
+
+where, the right-most-side of the equation has an additional parameter, :math:`a` to generalize the power of :math:`\lambda_k`.  For the inverse transform, there is an additional factor :math:`\frac{1}{\pi r}` in the state equation, and hence the integrand has :math:`\lambda_k` power, reduced by -1. While, for the 
+first-order hold approximation, the linear :math:`\epsilon` term increases :math:`\lambda_k` by +1. 
 
 
 Citation
 --------
-[1] `E. W. Hansen and P.-L. Law, "Recursive methods for computing the Abel transform and its inverse", J. Opt. Soc. A2, 510-520 (1985) <http://dx.doi.org/10.1364/JOSAA.2.000510>`_
+[1] `E. W. Hansen and P.-L. Law, "Recursive methods for computing the Abel transform and its inverse*, J. Opt. Soc. A2, 510-520 (1985). <http://dx.doi.org/10.1364/JOSAA.2.000510>`_
+
+[2] `E. W. Hansen, "Fast Hankel Transform", IEEE Trans. Acoust. Speech Signal Proc. 33, 666 (1985). <https://dx.doi.org/10.1109/TASSP.1985.1164579>`_
+
+[3] J. R. Gascooke, PhD Thesis: *"Energy Transfer in Polyatomic-Rare Gas Collisions and Van Der Waals Molecule Dissociation"*, Flinders University (2000).
```

Comment:
Should the rest of the titles also be in italics?

Reply:
The markup url quote inhibits any further markup. I have changed the Gascooke reference to link to the `https://github.com/PyAbel/abel_info` web page. It now looks similar.